src/uuid/json.cr

```diff
@@ -18,7 +18,7 @@ struct UUID
   # end
   #
   # example = Example.from_json(%({"id": "ba714f86-cac6-42c7-8956-bcf5105e1b81"}))
-  # example.id # => UUID(ba714f86-cac6-42c7-8956-bcf5105e1b81)
+  # example.id.hexstring # => "ba714f86cac642c78956bcf5105e1b81"
```

Comment:
What's the problem here? The example works fine as is.
https://carc.in/#/r/9thg

Reply:
In example codes like `CODE # => VALUE`, the part of `VALUE` must be evaluatable.
Because basically it is simply converted to `( CODE ).should eq( VALUE )` and validated in my testing app.

```crystal
# [invalid]
example.id # => UUID(ba714f86-cac6-42c7-8956-bcf5105e1b81)   # original code

# [valid candidates] 
example.id.hexstring # => "ba714f86cac642c78956bcf5105e1b81"           # I chose this one
example.id.inspect   # => "UUID(ba714f86-cac6-42c7-8956-bcf5105e1b81)" # Is this better?
p example.id         # UUID(ba714f86-cac6-42c7-8956-bcf5105e1b81)      # just a comment? (no validations)

# or put it into two lines?
example.id.class     # => UUID
example.id.hexstring # => "ba714f86cac642c78956bcf5105e1b81"
```

Reply:
Maybe the validation should take `(CODE.inspect).should eq("VALUE")` into account (as an alternative)? I've always thought of `# =>` to inspect the value. Such examples usally use simple primitives for demonstration and the inspect representation is typically identical to the source literal. But when a more complex value is inspected (like a `UUID`), the simple expression `CODE # => VALUE` no longer holds.

I'd leave this as is.

Reply:
Since there are no clear specification models for this verification, your idea of using `inspect` and `p!` is very appealing because it's simple and straightforward. 👍 

The key here is that, unlike eq-based verification, inspect-based verification requires conversion from natural text `VALUE` to crystal string `"VALUE"`. And this may not be easy to do, as it causes new problems such as double escaping of strings, the difference between `to_s` and `inspect`, and so on.

Since this is going to take some time, this PR discards the `UUID` and `HTTP::Status` modifications and doesn't validate those examples.

After that, I'm going to check the entire examples on the new inspect-based verifier base and then I'll put out another PR again. 😃